doc/mgr/dashboard.rst

```diff
@@ -359,16 +352,14 @@ To use the Object Gateway management functionality of the dashboard, you will
 need to provide the login credentials of a user with the ``system`` flag
 enabled.
 
-If you do not have a user which shall be used for providing those credentials,
-you will also need to create one::
+If you do not have an appropriate RGW user already, you will need to create one::
 
   $ radosgw-admin user create --uid=<user_id> --display-name=<display_name> \
       --system
 
-Take note of the keys ``access_key`` and ``secret_key`` in the output of this
-command.
+Take note of the keys ``access_key`` and ``secret_key`` in the output.
 
-The credentials of an existing user can also be obtained by using
+The credentials of an existing user can be obtained by using
```

Comment:
To obtain the credentials of an existing user...

Reply:
ack